standards/ICRC-3/ICRC-3.did

```diff
@@ -0,0 +1,101 @@
+type TxIndex = nat;
+
+type Account = record {
+    owner : principal;
```

Comment:
Shouldn't that be the address for privacy reasons? Or is it meant to be the address?

Reply:
In ICRC-1 ledgers, address is `(principal, subaccount)`; we decided to abandon the notion of `AccountId`.
Account identifiers don't give you much additional privacy and are a real pain to use in practice.
I explained my point of view in more detail here: https://mmapped.blog/posts/13-icp-ledger.html#account-id.